tests/Fixer/Annotation/RemovePHPStormAnnotationFixer/RemovePHPStormAnnotationFixerTest.php

```diff
@@ -19,7 +20,7 @@ public function test(string $filePath): void
 
     public function provideData(): Iterator
     {
-        yield self::yieldFiles(__DIR__ . '/Fixture');
+        return DataProviderFixtureYielder::yieldDirectory(__DIR__ . '/Fixture');
```

Comment:
The  `self::yieldFiles` actually only returns the 1st file, so it's a bug :bug: 

We have to fix it in ECS to return all files :+1:

Reply:
@samsonasik I won't be able to resolve this one in ECS, cause I have to work somewhere else,
could you look at it? :pray: 

Ideally with making a test so we know it works :D

Reply:
Thanks :clap: